src/main/java/org/dataone/hashstore/filehashstore/FileHashStore.java

```diff
@@ -112,6 +112,25 @@ enum HashStoreProperties {
         storePath, storeDepth, storeWidth, storeAlgorithm, storeMetadataNamespace
     }
 
+    enum HashStoreRefUpdateTypes {
+
+        add("add"), remove("remove");
+
+        final String refUpdateType;
+
+        HashStoreRefUpdateTypes(String updateType) {
+            refUpdateType = updateType;
+        }
+
+        public String getName() {
+            return refUpdateType;
+        }
+    }
```

Comment:
Thank you! I refactored this enum object, as well as other relevant enum objects and added javadocs. I also tried it on `DefaultHashAlgorithms`, but because the values cannot contain hyphens (`-`), it's not possible to reduce it any further.